custom/abdm/models.py

```diff
@@ -0,0 +1,32 @@
+import os
+import binascii
+
+from django.db import models
+
+
+class ABDMUser(models.Model):
```

Comment:
Assuming that these are mobile workers, why not use CommCareUser model only which already supports `HQApiKey`?

Reply:
To limit scope of token to ABDM APIs only. This was done as in future we want to develop more clients for similar APIs which do not need access to other HQ APIs.